src/index.ts

```diff
@@ -144,7 +144,7 @@ export default function fetchHAR(har: Har, opts: FetchHAROptions = {}) {
   }
 
   if ('postData' in request) {
-    if ('params' in request.postData) {
+    if (request.postData && 'params' in request.postData) {
```

Comment:
Kinda funky that TS wants you to check `request.postData` again here.

Reply:
yeah 😬 I wonder there's some minor detail about `in` that TS knows that we don't know here lol

Reply:
ts won't remember existence checks because side effects could change the value. You would want to throw postData into a variable at line 145 and then check that at 146, and use it at 147.